### PR TITLE
review-feature/menu

### PR DESCRIPTION
**Pull request by elawrobel**
Final menu [ new style and layout related with new design]

### CODE REVIEW

File: app/theme.js [hunk 1]
```diff
@@ -4,8 +4,13 @@ export const theme = {
     white: '#ffffff',
     blue: '#3b3aff',
     grey: '#ececea',
+    yellow: '#fde452',
+  },
+  fonts:{
+    title: 'Quicksand',
+    random: 'Roboto Slab',
+    beckup: 'sans-serif',
```
Comment (ucanthurtme) [inline app/theme.js:12]: typo

File: app/theme.js [hunk 2]
```diff
@@ -4,8 +4,13 @@ export const theme = {
     white: '#ffffff',
     blue: '#3b3aff',
     grey: '#ececea',
+    yellow: '#fde452',
+  },
+  fonts:{
+    title: 'Quicksand',
+    random: 'Roboto Slab',
```
Comment (ucanthurtme) [inline app/theme.js:11]: can we make it more descriptive ?

File: app/components/Menu/index.js [hunk 3]
```diff
@@ -48,12 +98,13 @@ function Menu() {
           </NavLink>
         </NavItem>
         <NavItem>
-          <NavLink to="/add-event">
+          <Button to="/add-event">
             <FormattedMessage {...messages.add} />
-          </NavLink>
+            {/* QA: kady messages wpada w span, da się temu jakoś zapobiec? */}
```
Comment (ucanthurtme) [inline app/components/Menu/index.js:103]: co masz na myśli ?

Comment (ucanthurtme) [reply app/components/Menu/index.js:103]: edit: ok ok czaje

Comment (ucanthurtme) [reply app/components/Menu/index.js:103]:
```
        <FormattedMessage {...messages.add}>
              {msg => <div>{msg}</div>}
            </FormattedMessage>
```

please read carefully [children as function](https://www.robinwieruch.de/react-pass-props-to-component)

Comment (elawrobel) [reply app/components/Menu/index.js:103]: Right! I'll check this! Thanks!

**Pull request merged**